parcels/field.py

```diff
@@ -68,13 +68,14 @@ def from_netcdf(cls, name, varname, datasets, **kwargs):
         """
         if not isinstance(datasets, Iterable):
             datasets = [datasets]
-        lon = datasets[0]['nav_lon'][0, :]
-        lat = datasets[0]['nav_lat'][:, 0]
```

Comment:
These removed lines in field.py are now breaking test_peninsula each time, in which a grid is created, saved, then re-loaded using NEMOgrid.from_file. I will try and fix this tomorrow. You're right @mlange05, this would not break if NEMOgrid was a proper subclass of NetCDFGrid.